appveyor.yml

```diff
@@ -5,6 +5,11 @@ environment:
 
 install:
 
+cache:
+  - nsissetup.exe -> appveyor.yaml
+  - NsProcess.zip -> appveyor.yaml
+  - thirdparty\windows -> thirdparty\fetch-thirdparty-deps.ps1

```

Comment:
Why only the native dependencies?  Nuget fails just as often with the .net dlls.